broker/setup_broker/lvk/setup_broker.sh

```diff
@@ -34,15 +36,31 @@ fi
 
 #--- GCP resources used directly in this script
 broker_bucket="${PROJECT_ID}-${survey}-broker_files"
+bq_dataset="${survey}"
 topic_alerts="${survey}-alerts"
+topic_storebigquery="${survey}-BigQuery"
```

Comment:
Lower case when naming resources.

```suggestion
topic_storebigquery="${survey}-bigquery"
```